packages/viewer/src/components/scene-tree/scene-tree.tsx

```diff
@@ -386,6 +389,33 @@ export class SceneTree {
     });
   }
 
+  /**
+   * Performs an API call that will select the items with indexes between the given bounds.
+   *
+   * @param lowerBound The smaller index of the set of rows to select.
+   * @param upperBound The larger index of the set of rows to select.
+   * @param viewer The viewer to select in.
+   */
+  @Method()
+  public async performMultiSelection(
+    lowerBound: number,
+    upperBound: number,
+    viewer: HTMLVertexViewerElement
+  ): Promise<void> {
+    const rowsToMultiSelect = this.rows.filter(
+      (row) =>
+        row?.index && row?.index <= upperBound && row?.index >= lowerBound
+    );
+    this.multiSelectedRows = rowsToMultiSelect;
+    if (viewer && rowsToMultiSelect) {
+      await rowsToMultiSelect.forEach((row) => {
+        if (row?.node?.id?.hex) {
+          selectItem(viewer, row?.node?.id?.hex, { append: true });
+        }
+      });
```

Comment:
Hypothetically, a user could select thousands of items, so this'd be a lot of requests going over the wire. @jareddellitt can we expose an RPC in STS to handle range selection?